packages/fluentui/docs/src/views/Introduction.tsx

```diff
@@ -1,10 +1,10 @@
-import * as _ from 'lodash'
-import * as React from 'react'
-import { NavLink } from 'react-router-dom'
-import { Flex, Header } from '@fluentui/react'
+import * as _ from 'lodash';
```

Comment:
Did you format all the files with prettier or is that going to happen randomly as they're edited as we go along?

Reply:
I am formatting with prettier as we go along for now - but I think near at the final import, we'll prettier the whole thing